migrations/2025-10-15-112824_add_invoice_paid_event_type/up.sql

```diff
@@ -0,0 +1,9 @@
+-- Your SQL goes here
+ALTER TYPE "EventType"
+ADD VALUE 'invoice_paid';
+
+ALTER TYPE "EventObjectType"
+ADD VALUE 'subscription_details';
+
+ALTER TYPE "EventClass"
+ADD VALUE 'subscriptions';
```

Comment:
Can we have `IF NOT EXISTS`, so that these migrations would be redo-able / re-runnable?